app/views/govuk_publishing_components/components/docs/error_message.yml

```diff
@@ -0,0 +1,14 @@
+name: Form error message
+description: Use error messages for any form fields.
+accessibility_criteria: |
```

Comment:
can you add the `govuk_frontend_components` to this, so that it'll link to the design system?

Reply:
Updated.